packages/rspack/src/builtin-loader/lightningcss/index.ts

```diff
@@ -254,7 +254,12 @@ export type LoaderOptions = {
 	targets?: Targets | string[] | string;
 	include?: FeatureOptions;
 	exclude?: FeatureOptions;
+	/**
+	 * @deprecated Use `drasfts` instead.
+	 * This will be removed in the next minor version.
```

Comment:
I prefer to remove it in the next major release, as it's a breaking change and is easy to maintain compatibility.

```suggestion
	 * This will be removed in the next major version.
```

Reply:
Done~